meshroom/ui/reconstruction.py

```diff
@@ -694,6 +694,33 @@ def allViewIds(self):
         """ Get all view Ids involved in the reconstruction. """
         return [vp.viewId.value for node in self._cameraInits for vp in node.viewpoints.value]
 
+    @Slot()
+    def clearImages(self):
+        """ Clear the list of viewpoints and intrinsics for the current CameraInit node. """
+        with self.groupedGraphModification("Clear Images"):
+            self.deleteIntrinsics(self._cameraInit.attribute("viewpoints"), self._cameraInit.attribute("intrinsics"))
+
+    @Slot()
+    def clearAllImages(self):
+        """ Clear the list of viewpoints and intrinsics for all the CameraInit nodes. """
+        with self.groupedGraphModification("Clear All Images"):
+            for cameraInit in self._cameraInits:
+                self.deleteIntrinsics(cameraInit.attribute("viewpoints"), cameraInit.attribute("intrinsics"))
+
+    def deleteIntrinsics(self, viewpoints, intrinsics):
+        """ Delete a set of provided viewpoints and intrinsics. """
+        # Delete all viewpoints
```

Comment:
Maybe better to use "resetValue" https://github.com/alicevision/Meshroom/blob/develop/meshroom/core/attribute.py#L358

Reply:
Done (and infinitely better!).